binary_transparency/firmware/devices/usb_armory/README.md

```diff
@@ -140,11 +141,31 @@ There are some invariants which must hold for this chain to work:
     following contents:
     * `armory-boot.conf` - a JSON file which tells the bootloader which files
       to load
-    * a valid ARM linux Kernel image
-    * a valid DTB file
-   Note that the `armory-boot.conf` file also contains SHA256 hashes of the
-   kernel and DTB files, and these MUST be correct.
+    * Either:
+        * to boot a Linux Kernel:
+           * a valid ARM linux Kernel image
+           * a valid DTB file
+        * to boot ELF unikernel: 
+           * a valid ARM bare-metal ELF binary/unikernel
 
+   Note that the `armory-boot.conf` file also contains SHA256 hashes of 
+   all files referenced, and these MUST be correct.A
+
+To aid in the creation of valid firmware images, the 
+`[cmd/usb_armory/image_builder/build.sh](/binary_transparency/firmware/cmd/usb_armory/image_builder/build.sh)`
+script can be used:
+
+```bash
+$ ./cmd/usb_armory/image_builder/build.sh -u ./testdata/firmware/usb_armory/example/tamago-example -o /tmp/armory.ext4
+
+/tmp/armory2.ext4: Writing to the journal is not supported.
```

Comment:
This says `armory2` and below, but the command does not have the `2`?

Reply:
:eye: you caught me, this was the Radio Edit.
done

Reply:
Looks like the error still remains on L161 (this line!).

Reply:
d'oh, done